src/main/java/com/powsybl/caseserver/CaseController.java

```diff
@@ -224,4 +225,13 @@ public ResponseEntity<List<CaseInfos>> getMetadata(@RequestParam("ids") List<UUI
         LOGGER.debug("get Cases metadata");
         return ResponseEntity.ok().body(caseService.getMetadata(ids));
     }
+
+    @GetMapping(value = "/cases/caseBaseName")
+    @Operation(summary = "Get case base name")
+    @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "case base name"), })
+    public ResponseEntity<String> getCaseBaseName(@RequestParam("caseName") String caseName) {
+        LOGGER.debug("getCaseBaseName request received");
+        String baseName = DataSourceUtil.getBaseName(caseName);
+        return ResponseEntity.ok().body(baseName);
```

Comment:
remove redundant body method call, it is unnecessary when returning the object directly
```suggestion
        return ResponseEntity.ok(baseName);
```